_posts/2020-02-06-pwa-cache-me-if-you-can.markdown

```diff
@@ -0,0 +1,245 @@
+---
+layout: post
+title: "PWA Cache me if you can"
+date: 2020-02-06
+tags: PWA react
+author: Natalie Laing
+image: img/code.jpg
+---
+
+
+### What is a PWA anyway?
+
+> This morning I went to write an article outlining a few tips about implementing a progressive web app (PWA). But when I went to introduce the topic, I again encountered what was the hardest part about the whole topic in the first place: I have a really hard time describing what a progressive web app actually is.
+
+
+![in the feels](/img/blog/pwa/feels.jpeg)
+
+The term PWA was coined in 2015, by designer Frances Berriman and Google Chrome engineer Alex Russell. 
+
+To create a PWA, you are going to need a service worker. A service worker is a script that is run in the background and acts as a network proxy between your app and the server to deliver that efficient caching strategy and offline capability PWA's are known for. 
```

Comment:
Personal opinion - you might want to describe the _what_ and the _why_ before the _how_?